docs/CreateAzureADforAzureBot.md

```diff
@@ -30,10 +30,14 @@ If you would like to authenticate against your own Azure AD for development or p
 ## 3. Configure the AAD application for use with the AzureBot
 Go to the configure tab of the application you just created and do the following:
 
- * Save the *Client ID* value. You will need this for the AzureBot development
- * Select duration for a key and click save.  Securely save the application secret key displayed on the screen - you will also need this for AzureBot Development.  You won't get access to this key again.
- * Add and save these two Reply URLs as follows: <br>https://_mycustomsite_.azurewebsites.com/api/OAuthCallback
- <br>https://localhost/api/OAuthCallback
+ * Save the *Client ID* value. This is the **ActiveDirectory.ClientId** for AzureBot development
+ * Select duration for a key and click save.  Securely save the application secret key displayed on the screen - This will be the **ActiveDirectory.ClientSecret** value used for AzureBot development..  You won't get access to this key again.
+ * Remove any existing Reply URLs and then add and save these two Reply URLs as follows: 
+   <br>https://localhost:3978/api/OAuthCallback 
```

Comment:
Should we add "This will be the ActiveDirectory.RedirectUrl value used for AzureBot development" here?